<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>environment</class>
 <widget class="QWidget" name="environment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="frame_A">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_title_A">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>냉동 창고 (A)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_temp_A">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 온도: -18.5°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_target_temp_A">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>설정 온도: -18.0°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>온도 설정 (-30~0°C):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_temp_A">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>-18.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_temp_A">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>온도 설정</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status_A">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #4CAF50; color: white; border-radius: 5px; padding: 2px;</string>
    </property>
    <property name="text">
     <string>정상</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mode_A">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2196F3; color: white; padding: 3px; border-radius: 3px;</string>
    </property>
    <property name="text">
     <string>냉각모드 작동중</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_B">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_title_B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>냉장 창고 (B)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_temp_B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 온도: 4.2°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_target_temp_B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>설정 온도: 4.0°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>온도 설정 (0~10°C):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_temp_B">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>4.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_temp_B">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>온도 설정</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status_B">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #4CAF50; color: white; border-radius: 5px; padding: 2px;</string>
    </property>
    <property name="text">
     <string>정상</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mode_B">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2196F3; color: white; padding: 3px; border-radius: 3px;</string>
    </property>
    <property name="text">
     <string>냉각모드 작동중</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_C">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_title_C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>상온 창고 (C)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_temp_C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 온도: 24.2°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_target_temp_C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>설정 온도: 24.0°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>온도 설정 (15~30°C):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_temp_C">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>24.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_temp_C">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>온도 설정</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status_C">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #4CAF50; color: white; border-radius: 5px; padding: 2px;</string>
    </property>
    <property name="text">
     <string>정상</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mode_C">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FF5722; color: white; padding: 3px; border-radius: 3px;</string>
    </property>
    <property name="text">
     <string>가열모드 작동중</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_D">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_title_D">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>비식품창고 (D)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_temp_D">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>현재 온도: 23.8°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_target_temp_D">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>설정 온도: 23.0°C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>온도 설정 (15~30°C):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_temp_D">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>23.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_temp_D">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>온도 설정</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status_D">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #4CAF50; color: white; border-radius: 5px; padding: 2px;</string>
    </property>
    <property name="text">
     <string>정상</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mode_D">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2196F3; color: white; padding: 3px; border-radius: 3px;</string>
    </property>
    <property name="text">
     <string>냉각모드 작동중</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
